docs/4-entry-points/4-guardians-registry.md

```diff
@@ -20,12 +20,41 @@ This module is in the one handling all the staking/unstaking logic for the guard
     - Save the guardian token address
     - Save the total active balance limit
 
-### 4.4.2. Activate
+### 4.4.2. Stake
 
-- **Actor:** Guardian of the Protocol
+- **Actor:** Guardian or an external entity incentivized in a guardian of the Protocol
```

Comment:
Perhaps we can replace "an external entity incentivized in a guardian of the Protocol" (which doesn't make much sense to me), to just "whitelisted relayer" for now?